contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -3285,15 +3285,44 @@ bbf_ProcessUtility(PlannedStmt *pstmt,
 								{
 									RoleSpec   *rolspec = lfirst(item);
 									char	   *user_name;
+									char       *db_principal;
```

Comment:
We should pfree `db_principal`. `db_owner_name` is a const char*, it can't be freed.

Reply:
We should not pfree db_principal as it is pointing to string literal.

Reply:
it is it pointing to string literal then make it const char * here itself. 

`const char * db_principal = drop_user ? "user" : "role"`